construction_impl.go

```diff
@@ -123,6 +127,23 @@ func formatAddress(network filAddr.Network, addr filAddr.Address) string {
 	return addr.String()
 }
 
+func ethToFilAddress(ethAddress [20]byte) (filAddr.Address, error) {
```

Comment:
I think you should also add a public func, like `EthToFilAddress(ethAdd string) (string, error)` and `EthToFilAddress(ethAdd string) (string, error)` . I think is what most people will use. Also, it should also work if the address has the "0x" prefix.
Maybe something like [this](https://github.com/Zondax/rosetta-filecoin-proxy/blob/ad4f819dab906b7bf356b8a739f378543bb279fe/rosetta/services/construction.go#L292) (we can then move that code from the proxy to here)

Reply:
Ok I can add that